lib/matplotlib/pyplot.py

```diff
@@ -1299,13 +1299,13 @@ def subplot2grid(shape, loc, rowspan=1, colspan=1, fig=None, **kwargs):
     -----
     The following call ::
 
-        subplot2grid(shape, loc, rowspan=1, colspan=1)
+        subplot2grid((nrows, ncols), (row, col), rowspan, colspan)
```

Comment:
This PR is an improvement, but while in the neighborhood I think it would make sense to go farther. The present docstring lacks a `Returns` block, which could be added.  With or without that, the lines changed here could be made more informative by prepending `ax = ` in the appropriate places.

Reply:
ok, I just copied the return spec of `subplot()` for now (and add `ax = ...`).

Reply:
Perfect, thank you.